bundles/org.eclipse.passage.loc.report.ui/src/org/eclipse/passage/loc/report/internal/ui/jface/user/PreviewPage.java

```diff
@@ -87,6 +87,9 @@ private String userInfo(UserDescriptor user) {
 	}
 
 	private String companyInfo(UserOriginDescriptor company) {
+		if (company == null) {
+			return "no information provided (not loaded)"; //$NON-NLS-1$
```

Comment:
I think this is user visible message and should be externalized

Reply:
Indeed. Fixed.